src/options/settings/components/KeyboardShortcuts.js

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react'
+import { Checkbox } from '../../../common-ui/components'
+import * as utils from 'src/content-tooltip/utils'
+const styles = require('./settings.css')
+
+class KeyboardShortcuts extends React.PureComponent {
```

Comment:
For this new module, please stick to our naming conventions (I know lots of existing stuff has weird mixed naming conventions, but we're trying to enforce it for all new modules):
https://github.com/WorldBrain/Memex/blob/develop/GETTING-STARTED.md#naming-conventions

Also please write new all modules as TypeScript modules rather than JS.

tl;dr: (although I really recommend all contributors read it) change it to `keyboard-shortcuts-container.tsx`